docs/examples/ipfs-101/main.go

```diff
@@ -152,30 +171,31 @@ func writeTo(nd files.Node, fpath string) error {
 /// -------
 
 func main() {
-	fmt.Printf("Starting")
+	fmt.Println("Starting")
+
+	if err := setupPlugins(""); err != nil {
```

Comment:
Isn't the path important?

Reply:
@aschmahmann ?

Reply:
Not for flatfs. We will (on Linux and maybe OSX?) load compiled plugins from the path. However, we'll also load all of the "preloaded" plugins, which include FlatFS and badger. 

https://github.com/ipfs/go-ipfs/blob/23ac8a22090be297cb4d0e4cf310372e47fa7f72/plugin/loader/preload.go#L14-L19

https://github.com/ipfs/go-ipfs/blob/23ac8a22090be297cb4d0e4cf310372e47fa7f72/plugin/loader/loader.go#L107-L111